ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -2420,10 +2432,11 @@ tests:
     cluster_profile: gcp-qe
     env:
       E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-ovnkubernetes and not @fips'
+      FEATURE_SET: TechPreviewNoUpgrade
       TAG_VERSION: '@4.12'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-gcp-ipi-xpn-techpreview
```

Comment:
For the workflow, QE is using `ipi-conf-feature-set` to set `techpreview`, but not `ipi-conf-techpreview`, it only supported in 4.12+ jobs, so actually we do not need the update.

Reply:
The changes that I've made supersede `ipi-conf-feature-set`, I'm trying to make the usage of the feature sets consistent across our jobs so even though you weren't using `ipi-conf-techpreview`, I've moved this over as well.

The idea is that you won't need specific workflows for techpreview, but just add the env var to an existing workflow and you are done, it should mean a lot of deduplication in the future

Reply:
Yeah, I see. QE will follow up the same idea, and remove `ipi-conf-feature-set` together in https://github.com/openshift/release/pull/39002 to avoid the duplicates.